src/server.ts

```diff
@@ -73,6 +73,7 @@ async function startServer() {
     ) {
       // ignore timeouts and terminated connections
     } else {
+      console.error(err.message)
```

Comment:
was this intentional? just in case :P

Reply:
yes, i noticed that when uncaught errors occurred they didn't show up on the terminal without that (in my case I was trying to start the server in one terminal and there as another one already running on that same port, and it would die without logging anything)